unlock-app/src/__tests__/components/creator/CreatorLockForm.test.js

```diff
@@ -56,6 +56,20 @@ describe('CreatorLockForm', () => {
       .forEach(error => expect(resetError).toHaveBeenCalledWith(error))
   }
 
+  describe('things that should not fail', () => {
+    it('properly handle unlimited keys when editing', () => {
+      const wrapper = makeLockForm({ maxNumberOfKeys: 0, convert: true })
+
+      const submit = wrapper.getByText('Submit')
+      expect(submit).not.toBeNull()
```

Comment:
FYI I know we have been doing this assertion in a number of places but `getByText` throws if the component is null